pkgs/tools/misc/ethminer/default.nix

```diff
@@ -8,29 +9,30 @@
   boost,
   makeWrapper,
   cudatoolkit,
+  withCuda ? cudatoolkit != null,
   mesa,
   ethash,
   opencl-info,
   ocl-icd,
   openssl,
   pkg-config,
   cli11
-}:
+}@args:
```

Comment:
Why do we need args here?

Reply:
Infinite recursion in let binding later.